test/parallel/test-stream2-readable-wrap.js

```diff
@@ -63,7 +62,7 @@ function runTest(highWaterMark, objectMode, produce) {
   }
 
   const w = new Writable({ highWaterMark: highWaterMark * 2,
-                           objectMode: objectMode });
+                           objectMode });
```

Comment:
This is the second exception to the described rule, again, feel free to request changes here.